mlrun/runtimes/sparkjob.py

```diff
@@ -340,19 +341,17 @@ def _resolve_crd_object_status_info(
         # it is less likely that there will be new stable states, or the existing ones will change so better to resolve
         # whether it's a transient state by checking if it's not a stable state
         state = crd_object.get("status", {}).get("applicationState", {}).get("state")
-        in_transient_state = state not in ["COMPLETED", "FAILED"]
-        desired_run_state = None
+        in_transient_state = state not in SparkApplicationStates.stable_states()
```

Comment:
```suggestion
        in_progress = state not in SparkApplicationStates.stable_states()
```

Reply:
move the discussion on this to previous thread